### ISSUE TITLE
Question about perceptual loss

### ISSUE THREAD

**Issue by jxust01** [shregar1/SKETCH_TO_ANIME#1]
the x_content and x_style in perceptual_loss function both equal to y as 
trianer.py line 77
```
perceptual_loss = self.P_Loss.find(y, y, y_fake)
```
why not use (y, y_fake) as input, and redefine the perceptual_loss function?
